### PR TITLE
Refactored consent display

### PR DESCRIPTION
**Pull request by buske**
- Refactored HTML, CSS, JS to remove some redundancies and make class names clearer
- Updated translation strings

### CODE REVIEW

**Comment by buske** [comment]
Nominating @allasm and @itaiGershtansky for this small refactoring PR.

**Comment by itaiGershtansky** [comment]
+1

File: components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml [hunk 1]
```diff
@@ -702,27 +715,26 @@
 $xwiki.jsx.use('PhenoTips.PatientConsent')
 $xwiki.ssx.use('PhenoTips.PatientConsent')
 #if ($services.consentauthorizer.consentsGloballyEnabled())
-  (% id='consents-block' class='$xcontext.action' %)(((
+  #set($consentBaseUrl = "/rest/patients/${doc.getName()}/consents/")
+  (% id='consents-block' class='$xcontext.action' data-url='$consentBaseUrl' %)(((
     {{html clean='false' wiki='false'}}&lt;div class='patient-id hidden'&gt;${doc.getName()}&lt;/div&gt;{{/html}}
-    (% class='view-title hidden' %)(((
+    (% class='consents-title' %)(((
```
Comment (allasm) [inline components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml]: Looks ok as is but want confirm that CSS rule for "consents-title" is omitted on purpose

**Comment by allasm** [comment]
+1

File: components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml (hunk 1 above)
Comment (buske) [inline components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml]: Yep, it was on purpose. There weren't any styles associated with that class (nor `view-title` before). Nice spot, though!

**Comment by buske** [comment]
@sdumitriu: ready for your final review

**Pull request merged**